workspace/test_configurations.go

```diff
@@ -0,0 +1,74 @@
+package workspace
+
+import "runtime"
+
+type TestConfiguration struct {
+	// The static portion of the test Command, which will be run for every test on this track. Examples include `cargo test` or `go test`.
+	// Might be empty if there are platform-specific versions
+	Command string
+
+	// Windows-specific test command. Mostly relevant for tests wrapped by shell invocations. Falls back to `Command` if we're not running windows or this is empty.
+	WindowsCommand string
+
+	// Some tracks test by running a specific file, such as `ruby lasagna_test.rb`. Set this to `true` to look up and include the name of the default test file(s).
+	AppendTestFiles bool
+	// All args after `--` aren't parsed and are passed to the test command. Some languages (especially `rust`) expect an additional `--` between _their_ args. So instead of requiring a user to call `exercism test -- -- --include-ingored` to run all `rust` tests, set this to `true` to separate the args passed to the test runner by a `--` automatically.
+	AutoSeparateArgs bool
```

Comment:
Couldn't this just be baked into the test command? `cargo test --` seems to work fine.

```
"rust": {
  Command:          "cargo test --"
}
```

Reply:
yeah, that's a great point. I think `rust` was the only track that cared about this, so that's probably simpler than having a struct field for it